components/raftstore/src/store/fsm/peer.rs

```diff
@@ -3946,6 +3984,15 @@ where
         let leader_id = self.fsm.peer.leader_id();
         let request = msg.get_requests();
 
+        if self.fsm.peer.force_leader {
```

Comment:
Do we need a test for this?

Reply:
it's already covered by the test, will add a comment in the test